.travis.yml

```diff
@@ -12,5 +12,7 @@ branches:
     - master
 
 install:
-  - npm install electron -g
-  - npm install
\ No newline at end of file
+  - npm install yarn electron webpack typescript -g
```

Comment:
What node/npm version are you running? Npx is included in npm v6 and above. No global installations should be required for supported nodejs engines.

Reply:
I'll replace travis-ci with github actions, so I'll just delete it.